libraries/config/src/genesis.cpp

```diff
@@ -129,4 +129,11 @@ bytes GenesisConfig::rlp() const {
 
 blk_hash_t GenesisConfig::genesisHash() const { return dev::sha3(rlp()); }
 
+std::pair<uint64_t, uint64_t> GenesisConfig::getGasLimits(uint64_t block_number) const {
+  if (this->state.hardforks.isOnCornusHardfork(block_number)) {
```

Comment:
we don't use this-> anywhere in the codebase as far as I know

Reply:
Removed this